mk_verificator/tests/ceph/test_ceph.py

```diff
@@ -0,0 +1,11 @@
+from mk_verificator import utils
+
+
+def test_check_ceph_osd(local_salt_client):
+    config = utils.get_configuration(__file__)
+    osd_fail = \
```

Comment:
e.g. output of the test
E       AssertionError: Some osds are in down state
E       assert not 'XX  0.XXXX         osd.XX       down        0          1.00000'